assets/vqganclip.yaml

```diff
@@ -0,0 +1,25 @@
+---
+
+- type: model
+  name: VQGAN-CLIP
+  organization: EleutherAI
+  description: VQGAN-CLIP is a model that better generates and edits images using
+    a multimodal encoder to guide image generation.
+  created_date: 2023-09-04
+  url: https://arxiv.org/pdf/2204.08583.pdf
+  model_card: ''
+  modality: text input with image output
+  analysis: Evaluated by human testers rating alignment of text input, image output
+    pairs.
+  size: 227M parameters
+  dependencies: [VQGAN, CLIP]
+  training_emissions: unknown
+  training_time: Less than 1 V100-hour
```

Comment:
Is this right, this seems like extremely little time for a single GPU, would be very surprised

Reply:
The paper says that it took 0 V100-hours to train, which I was also surprised by. I'll check again though to see if I'm missing anything.